sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -2277,7 +2280,7 @@ async def hack_iter_next_mock_error(self):
                 assert receiver.error_raised
                 assert receiver.execution_times >= 4  # at least 1 failure and 3 successful receiving iterator
 
-    @pytest.mark.skip(reason="TODO: iterator support")
+    @pytest.mark.skip(reason="TODO: Pyamqp Message Header Time to Live")
```

Comment:
Is this just because the attribute name changed?

Reply:
Yes let me fix and uncomment